xExtension-YouTube/extension.php

```diff
@@ -111,10 +111,21 @@ public function getHeight()
      *
      * @return bool
      */
-    public function isShowContent()
+    public function getShowContent()
```

Comment:
(nitpick) `isShowContent` was fine to me (but I don't really care)

Reply:
Done.